### DISCUSSION TITLE
New Faction: The Searchers

### DISCUSSION THREAD

**Discussion by mOctave** [endless-sky/endless-sky#7283] [Ideas]
I've had an idea for a little while of a new faction I would like to see in Endless Sky, or possibly a plugin if there's interest but it doesn't fit with the current lore. I would love to work on this either way, especially if someone else is interested with more artistic skill than I have. As far as I know, the just north of the space between the Deep and the Far North is still empty, so I would propose to put my (as of yet unnamed) species either here or to the west of the Deep.

# Goals
- Create an r-strategist species in a galaxy of K-strategists
- Create a species that isn't what it appears to be at first glance
- Create a species that only lives for about a decade
- Create more background for the Pug and Quarg
- Fill in more of the galaxy
- To create more sizes of graviton engines so that we don't only have (Medium Graviton Thruster/Steering)

# History
At some point in the past several thousand years, the Pug and Quarg saw a species just beginning to develop space-travel capabilities and disagreed on what to do with them, whether to expose them to humanity once both races had developed hyperspace capabilities (the Pug's opinion), or to keep them in their own space and let them develop in peace (the Quarg's opinion). The first option would allow both species to learn from each other and mature through conflict, but the second would be much safer for all involved. Both sides started to work toward their respective goals, but once the Pug started actively manipulating the hyperspace links to achieve their goal, the Quarg decided that things had gone too far, and attacked the Pug. The war ultimately ended in a stalemate: the new species was kept in their own region of space, but several Pug and Quarg ships were lost in space, eventually to be discovered by the new species. This advanced technology has provided the species both with a lot of power and the knowledge that somewhere in the cosmos an alien species waits.

# Technology
The species has graviton engines, although they are nowhere near as efficient as those used by the Quarg. They also have fairly advanced weapons and systems, but they tend to be very inefficient when it comes to size and so their ships are forced to compensate by being large, slow, and having high crew requirements. The species has a few captured jump drives (around twenty in total), but they have not been able to produce new ones because the technology that goes into them is so far in advance of their own as to seem magical.

# Viewpoint and Culture
The species seeks to bring order to the universe and gain a fuller understanding of everything around them. This is not entirely an "idiosyncratic hierarchy" idea, but more a reaction to the dangerous and chaotic environment they began their existence in. The species is a slightly r-strategist species (producing a lot of young with the hope that some will survive), but as they no longer have any natural predators, they are facing some overpopulation problems, and there is a strict caste system. Many lower-caste members of the short-lived species head for the stars, where they can get decent pay aboard the great ships that keep their planets running. The officers of the ships are almost always higher-caste. At first glance, the species appears to be made up entirely of pacifists who seek to outsmart their opponents instead of fight them, but that is not actually the case.

# Geography and Politics
The species is ruled from their home world. From that system, the Pug have opened wormholes to four other systems to the galactic north, east, south, and west respectively. The wormholes require stabilization similar to those in the Ember Wastes, but the key is different. From those four systems,a ring of systems is formed, each with a different specialization. Because most ships aren't allowed to use the wormholes, this means that various commodities are very expensive on one side of the ring and very cheap on the other in a predictable pattern. The systems are all named after elementary particles, because the species likes to name the very largest things in their lives after the very smallest. The vast majority of the species' population lives in the worlds of the ring, because their home world was not particularly hospitable.

The government is an oligarchy with complete power and their own agenda. Only the highest-caste citizens have any real say over what happens in their lives, and the commodity prices are controlled for profit, not for the good of the people. That said, the government is decisive and not overly oppressive, staying out of the lives of ordinary citizens in any noticeable way, and they provide stability even in challenging times, practising enlightened despotism.

The species has a strict first-contact policy. Newcomers to their space are to be welcomed, as much information about their technology should be discovered as possible, and they should be warned that they are not allowed to scan the species' ships, nor are they allowed to attempt to enter the capital system.

Concept art map:
![New Species](https://user-images.githubusercontent.com/73318970/193170181-ca759359-62c2-420c-ae62-90e1719c89fd.png)

# The Secret
The species' ships appear at first glance to be utterly unarmed, but they are not, and the high-caste officers have orders to destroy anyone who attempts to scan them and find out about their concealed weapons. These weapons are also only available in the capital (home) system, which the player can only access by boarding a ship that has access to it and stealing the wormhole key, as it is too far away by jump drive.

The capital system is also the only system with a shipyard. New ships, when purchased, are escorted out of the system and then have their key removed so that they cannot get back in.

# Ship Design
- High crew requirements
- Generally large
- Carry unmanned "research" and "mining" drones that have pretty heavy weaponry installed
- Look sort of like a cross between Arach ships and the Quarg Skylark? Open to suggestions here
- I find it odd how (normal) humans don't have ships the size of the Heron when they have far greater resources than the Remnant. This race does.

**Comment by MasterOfGrey** [comment]
So first comment of some important notes straight up:  
- having it accessible by jumping off the Wanderers or off the Far North: A-OK.
- having it accessible by jumping off the Deep: No - the explicit lore intent is that the Deep is too isolated for that.

As for first contact, please consider carefully how successful communication during first contact comes about.

Other than those points, this sounds very promising. :)

**Comment by DarcyManoel** [comment]
Some-such like this? (imagine links to inner system are wormholes)

<img width="801" alt="Screen Shot 2022-09-30 at 4 15 46 pm" src="https://user-images.githubusercontent.com/32188044/193207775-3686788f-20b2-48ba-8206-82cec1b3ea82.png">

**Comment by mOctave** [comment]
> having it accessible by jumping off the Deep: No - the explicit lore intent is that the Deep is too isolated for that.

Okay. That makes sense. It would probably fit much easier where @DarcyManoel shows it, and satisfy the lore requirements, although I would like to make sure that no one can access the centre system and so either jump drive range needs to be reduced in the ring systems (which seems a little odd to me) or it needs to be spread out (maybe adding more systems in the ring).

> As for first contact, please consider carefully how successful communication during first contact comes about.

Good point. I've been thinking about this a little, and I think first contact works more or less because they've _known_ for certain that there are alien species out there for thousands of years, and have been seeking to contact them. They've had a lot of time to plan out how to meet other races, and so have various translators, and a set of rules on what to do in a variety of first contact situations. If the player enters their systems, the player is hailed and, after a few minutes of difficult conversation, provides the species' translators with enough of a grasp of the human language that, after that point, communication is not much of a barrier. The player is the first "alien" the species has ever met, but they are more technologically advanced than humanity across the board (maybe Tier 1.5-2 overall) and have the benefit of a decisive government that can adapt quickly.

One other point: I went hunting on the assets repo for the graviton engine models but couldn't find them. Am I missing something, or does the repo not have the graviton models/gimp files?

**Comment by mOctave** [comment]
To fit with the Deep access rules and to avoid overlap with the Thi-Ruun, the faction (which I'm calling the Searchers, at least for now), is located here:
<img width="523" alt="Screen Shot 2022-10-09 at 1 13 18 PM" src="https://user-images.githubusercontent.com/73318970/194777562-3392644b-3582-4251-838b-d9b1af4111ba.png">

This is out of jump range of all human systems except Gamma Corvi, Orbona, and _possibly_ Limen (although I think Limen is just outside jump range).

Available as a fork of Endless Sky here: https://github.com/mOctave/endless-sky/tree/feat(race)-1

**Reply by mOctave** [reply]
Apparently this is still too close to the Deep, it will probably be moved in the future.

**Comment by roadrunner56** [comment]
This looks really interesting!

**Comment by mOctave** [comment]
Since most communication about the Searchers is happening over Discord now, I'll post the link to my server for them here: https://discord.gg/j6KqfFBsSp

**Comment by RisingLeaf** [comment]
you might also want to link the plugin

**Reply by mOctave** [reply]
I don't actually have a repo for the plugin, I'm just dropping zips into the Discord for now. I do have a fork, which I linked above.

**Comment by nomiga-ww** [comment]
这是来自QQ邮箱的假期自动回复邮件。你好，我最近正在休假中，无法亲自回复你的邮件。我将在假期结束后，尽快给你回复。
